backend/app/server/routes/post.py

```diff
@@ -17,10 +17,11 @@
 
 router = APIRouter()
 
+
 @router.post("/create", response_description="Post added into the database")
```

Comment:
Now the route is returning status code 200 which is "OK". It is more preferable to return status code 201 which indicates successfully created